src/index.js

```diff
@@ -1,17 +1,20 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import './index.css'
+import App from './App'
+import * as serviceWorker from './serviceWorker'
+import { BrowserRouter as Router } from 'react-router-dom'
```

Comment:
In your App.js you import `BrowserRouter as Switch`, here you import it as `Router` which is the right way to do. You don't need the BrowserRouter component in your App.js when you already wrap your index.js with it. Please have a look at the documentation again :)